### PR TITLE
fix version field

### LINKED ISSUE #138: Incorrect version field
**What happened**:
image
pixiu is now v0.2.0 version, but the version field is still 0.1.0

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

### PR DESCRIPTION
**Pull request by lbh2001**
**What this PR does**:
Fixed version field.

**Which issue(s) this PR fixes**:

Fixes #138 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by AlexStocks — APPROVED** [review]

File: cmd/pixiu/pixiu.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ import (
 )
 
 // Version pixiu version
-var Version = "0.1.0"
+var Version = "0.2.0"
```
Comment (AlexStocks) [inline cmd/pixiu/pixiu.go:43]: 0.3.0?

**Review by AlexStocks — APPROVED** [review]

File: cmd/pixiu/pixiu.go (hunk 1 above)
Comment (xiaoliu10) [reply cmd/pixiu/pixiu.go]: maybe v0.2.1 can merge this PR

**Review by xiaoliu10 — APPROVED** [review]

File: cmd/pixiu/pixiu.go (hunk 1 above)
Comment (lbh2001) [reply cmd/pixiu/pixiu.go]: @xiaoliu10  Sorry for my bothering, but I've sent something about [ospp](https://summer.iscas.ac.cn/#/org/orgdetail/dubbogo) program to your email, and there is also an issue I've created about that. Could you please take a look? Thank you so much! :blush:

**Comment by kezhenxu94** [comment]
I'd rather extract the variable as a build argument, otherwise, this kind of issue is highly possible to occur next time

**Comment by lbh2001** [comment]
> I'd rather extract the variable as a build argument, otherwise, this kind of issue is highly possible to occur next time

So we should put variables like this into config file in the future, right? Should I fix it?

**Comment by kezhenxu94** [comment]
> > I'd rather extract the variable as a build argument, otherwise, this kind of issue is highly possible to occur next time
> 
> So we should put variables like this into config file in the future, right? Should I fix it?

What I mean is a build argument, not configuration file, take the below as an example

https://github.com/apache/skywalking-cli/blob/d06d1ebf565756885650061ddd2e1fe527417052/cmd/swctl/main.go#L43

https://github.com/apache/skywalking-cli/blob/d06d1ebf565756885650061ddd2e1fe527417052/Makefile#L18

https://github.com/apache/skywalking-cli/blob/d06d1ebf565756885650061ddd2e1fe527417052/Makefile#L40

https://github.com/apache/skywalking-cli/blob/d06d1ebf565756885650061ddd2e1fe527417052/Makefile#L71

**Comment by kezhenxu94** [comment]
A release manager should take care of the build argument when drafting a new release, and that doesn't require code changes

**Comment by lbh2001** [comment]
> A release manager should take care of the build argument when drafting a new release, and that doesn't require code changes

I've understood what you mean, and I will try my best to implement it after learning about Makefile. But before that, I wonder why one of the checks didn't pass and how to fix it. (Maybe this question is ridiculous :cry:)

**Comment by kezhenxu94** [comment]
> Maybe this question is ridiculous

It's totally ok, there is no such question as ridiculous in community. Don't be panic

**Pull request merged**